qemu/tests/cfg/block_stream.cfg

```diff
@@ -21,6 +21,22 @@
     kill_vm = yes
     alive_check_cmd = dir
     tmp_dir = /tmp
+    Windows:
+        create_command = "D:\coreutils\DummyCMD.exe FILE 1073741824 1"
```

Comment:
What is "D" ? I think  you mean "WIN_UTILS".   it is better not to use fixed driver letter.

Reply:
@yanan-fu Yes, updated with WIN_UTILS, thanks.